src/modules/selectors.js

```diff
@@ -0,0 +1,5 @@
+export const enterBtn = document.querySelector('.enter');
+export const input = document.querySelector('.inputField');
+export const listblock = document.querySelector('.listBlock');
+export const clear = document.querySelector('.clear');
+export const refresh = document.querySelector('.refresh');
```

Comment:
- [ ] To improve readability it would be great if you could maintain the const declarations in the same file they are being used.